### PR TITLE
fix: remove esbuild + remove some any

### PR DESCRIPTION
**Pull request by ratik**
As appeared esbuild has some weird issues with TS parsing: like [this one](https://github.com/aelbore/esbuild-jest/issues/57) 

So replace esbuild with babel

 
https://github.com/neutron-org/neutron-tests/actions/runs/3903229531

### CODE REVIEW

**Review by sotnikov-s — APPROVED** [review]

**Review by foxpy — CHANGES_REQUESTED** [review]

File: src/helpers/wait.ts [hunk 1]
```diff
@@ -16,7 +17,7 @@ export const getRemoteHeight = async (sdk: any) => {
   return +block.data.block.header.height;
 };
 
-export const waitBlocks = async (sdk: any, n: number) => {
+export const waitBlocks = async (sdk: CosmosSDK, n: number) => {
```
[the remark this answers is not in the document]
Comment (ratik) [reply src/helpers/wait.ts:20]: fixed

**Review by sotnikov-s — DISMISSED** [review]

**Review by sotnikov-s — DISMISSED** [review]

File: src/testcases/interchaintx.test.ts [hunk 2]
```diff
@@ -318,22 +318,23 @@ describe('Neutron / Interchain TXs', () => {
         });
       });
       test('delegate after the ICA channel was closed', async () => {
-        let rawLog: any;
+        let rawLog: string;
```
[the remark this answers is not in the document]
Comment (albertandrejev) [reply src/testcases/interchaintx.test.ts:321]: btw, please replace `expect` call below with something similar to `expect(rawLog).stringContaining('no active channel for this owner')`

[the remark this answers is not in the document]
Comment (ratik) [reply src/testcases/interchaintx.test.ts:321]: So this way you'll have to `expect(rawLog || '').stringContaining('no active channel for this owner')` as `stringContaining` accepts string only. Let's leave it as it is

**Review by albertandrejev — DISMISSED** [review]

File: src/helpers/cosmos.ts [hunk 3]
```diff
@@ -924,21 +926,26 @@ export class CosmosWrapper {
...
     return await this.queryContract<SingleChoiceProposal>(propose_contract, {
       proposal: {
         proposal_id: proposalId,
       },
     });
   }
 
-  async queryTotalVotingPower(core_contract: string): Promise<any> {
+  async queryTotalVotingPower(
+    core_contract: string,
```
Comment (NeverHappened) [inline src/helpers/cosmos.ts:938]: can we change all arguments in cosmos.ts to camelCase? I've also seen `propose_contract` arg.

**Review by NeverHappened — CHANGES_REQUESTED** [review]

**Review by foxpy — APPROVED** [review]

File: src/helpers/cosmos.ts (hunk 3 above)
Comment (ratik) [reply src/helpers/cosmos.ts:938]: but actually this one is out of scope of this pr. will do one more

**Review by albertandrejev — DISMISSED** [review]

**Review by NeverHappened — DISMISSED** [review]

**Review by foxpy — APPROVED** [review]
utACK

File: src/helpers/cosmos.ts [hunk 4]
```diff
@@ -841,7 +843,7 @@ export class CosmosWrapper {
 
   async checkPassedProposal(propose_contract: string, proposalId: number) {
     await getWithAttempts(
-      this,
+      this.sdk,
```
Comment (foxpy) [inline src/helpers/cosmos.ts:846]: whaaat, how does it work if this function accepts `CosmosWrapper`, not `CosmosSDK`????

**Review by foxpy — CHANGES_REQUESTED** [review]
Requesting changes because this looks very bad.

File: src/helpers/cosmos.ts (hunk 4 above)
Comment (ratik) [reply src/helpers/cosmos.ts]: can't remember it 🤡
but the idea looks nice

**Review by foxpy — APPROVED** [review]
Looks good

**Review by oopcode — APPROVED** [review]

**Pull request merged**